take_pic.py

```diff
@@ -0,0 +1,38 @@
+"""
+Takes pictures of the user then converts them to encodings and saves them
+
+press q to quit
+and p to take a photo then enter name
+"""
+import sys
+import pickle
+import cv2
+import face_recognition
+CAMINDEX = 0
+#get webcam
+cap = cv2.VideoCapture(CAMINDEX)
```

Comment:
One very likely and common scenario in running this script would be if the camera isn't actually connected to the device. Consider using a try/catch around this line, printing a nice human message if a camera isn't present and then using sys.exit(1)